src/middleware/builtin.ts

```diff
@@ -210,30 +211,40 @@ export function matchConstraints(
  * Middleware that filters out messages that don't match pattern
  */
 export function matchMessage(
-  pattern: string | RegExp,
+  pattern: string | RegExp | (string | RegExp)[],
 ): Middleware<SlackEventMiddlewareArgs<'message' | 'app_mention'>> {
   return async ({ event, context, next }) => {
-    let tempMatches: RegExpMatchArray | null;
-
+    let tempMatches: RegExpMatchArray | null = null;
+    let pattArray: (string | RegExp)[] = [];
```

Comment:
To follow the code style, I would spell out the word (they seem to prefer more verbose variable names).  patternArray ?

Reply:
Fixed, in accordance with this suggestion.